docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -26,16 +19,16 @@ endif::[]
 For production environments, the deployment method will likely be
```

Comment:
Can we elaborate on how the deployment method is different?

Reply:
I think we should have an `ifeval` here for downstream content, and instead of this sentence we should have "Install the OpenStack Operators according to section 1 of the deployment guide." and link to https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0-beta/html-single/deploying_red_hat_openstack_services_on_openshift/index#assembly_installing-and-preparing-the-Operators